src/test/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketMockApiFactory.java

```diff
@@ -2,15 +2,16 @@
 
 import com.cloudbees.jenkins.plugins.bitbucket.api.BitbucketApi;
 import com.cloudbees.jenkins.plugins.bitbucket.api.BitbucketApiFactory;
-import com.cloudbees.plugins.credentials.common.StandardUsernamePasswordCredentials;
+import com.cloudbees.jenkins.plugins.bitbucket.api.BitbucketAuthenticator;
 import edu.umd.cs.findbugs.annotations.CheckForNull;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import edu.umd.cs.findbugs.annotations.Nullable;
 import hudson.Extension;
 import hudson.ExtensionList;
+import org.apache.commons.lang.StringUtils;
+
```

Comment:
Check your IDE settings for import sorting... it's all of them not just the two I have tagged